examples/unreliable-datagram/README.md

```diff
@@ -0,0 +1,42 @@
+# Unreliable Datagram
+
+This folder contains an example of sending and receiving an 'unreliable datagram' in `s2n-quic`. An 'unreliable datagram' is data that is not re-transmitted upon packet loss, and is defined in https://datatracker.ietf.org/doc/rfc9221. This is in contrast to 'stream data' which is re-transmitted during packet loss. Stream data is the preferred method of sending application data, as it comes with support for features like congestion-controllers and packet-retransmission. Additionally, unreliable datagram users are responsible for making sure their datagrams fit into the packet space given, as the RFC does not support data re-assembly. A common use-case for unreliable datagrams is video or audio streaming, where individual datagrams are small and some loss is acceptable.
```

Comment:
* [Retransmit is a word](https://www.merriam-webster.com/dictionary/retransmit)
* Datagrams are congestion controlled but not flow controlled

```suggestion
This folder contains an example of sending and receiving an 'unreliable datagram' in `s2n-quic`. An 'unreliable datagram' is data that is not retransmitted upon packet loss, and is defined in https://datatracker.ietf.org/doc/rfc9221. This is in contrast to 'stream data' which is retransmitted during packet loss. Stream data is the preferred method of sending application data, as it comes with support for features like flow control and packet retransmission. Additionally, unreliable datagram users are responsible for making sure their datagrams fit into the packet space given, as packet fragmentation/assembly is not allowed. A common use-case for unreliable datagrams is video or audio streaming, where individual datagrams are small and some loss is acceptable.
```

Reply:
Thanks! I tend to go overboard with my use of dashes :)